lib/model/token.dart

```diff
@@ -29,33 +27,38 @@ class Token {
       if (model.tokenType != null) {
         ret["token_type"] = model.tokenType;
       }
-      if (model.refreshToken != null ) {
+      if (model.refreshToken != null) {
         ret["refresh_token"] = model.refreshToken;
       }
-      if (model.expiresIn != null ) {
+      if (model.expiresIn != null) {
         ret["expires_in"] = model.expiresIn;
       }
-      if (model.expireTimeStamp != null ) {
+      if (model.expireTimeStamp != null) {
         ret["expire_timestamp"] = model.expireTimeStamp.millisecondsSinceEpoch;
-      }    
+      }
     }
     return ret;
   }
 
   static Token fromMap(Map map) {
-    if (map == null)
-      throw new Exception("No token from received");
+    if (map == null) throw new Exception("No token from received");
     //error handling as described in https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#error-response-1
-    if ( map["error"] != null )
-      throw new Exception("Error during token request: " + map["error"] + ": " + map["error_description"]);
+    if (map["error"] != null)
+      throw new Exception("Error during token request: " +
+          map["error"] +
+          ": " +
+          map["error_description"]);
 
     Token model = new Token();
     model.accessToken = map["access_token"];
     model.tokenType = map["token_type"];
-    model.expiresIn = map["expires_in"];
+    model.expiresIn = int.tryParse(map["expires_in"]) ?? 60;
```

Comment:
<img width="388" alt="Screen Shot 2019-06-28 at 11 11 39 AM" src="https://user-images.githubusercontent.com/14207417/60362358-932eb780-9995-11e9-877f-c19894d3fdba.png">

@Sisco21  after your changes when tried to run on example application  existing in this repo i am getting error shown in above image.

Reply:
@yugandh  Did you still got the error? I can't reproduce with our tenant.

Best regards.

Reply:
it's looks good. Functionality is working.